components/brave_shields/resources/adblock_bundle.grd

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grit latest_public_release="0" current_release="1" output_all_resource_defines="false">
+  <outputs>
+    <output filename="grit/adblock_bundle.h" type="rc_header">
+      <emit emit_type='prepend'></emit>
+    </output>
+    <output filename="adblock_bundle.pak" type="data_package" />
+  </outputs>
+  <release seq="1">
+    <includes>
+      <!-- Brave Shields preloaded component data -->
+      <include name="IDR_BRAVE_ADBLOCK_LIST_CATALOG" file="../../../third_party/adblock-resources/src/filter_lists/list_catalog.json" type="BINDATA" />
```

Comment:
Until it is .json, let's make it gzipped.

Reply:
that will reduce the size on disk, but increase the processing when is gets read

Reply:
if I'm not mistaken it's a general suggestion for text-like data. CPU overhead is minimum.
The exclusion: if you need store the raw data as a pointer (from a memory mapped .pak file).

Reply:
I'm not against it necessarily, just pointing out that there is some overhead to unzip it

Reply:
https://github.com/brave/brave-core/pull/22188/files#r1508260508